cypress/page/login.page.ts

```diff
@@ -0,0 +1,18 @@
+class Login {
+    private static emailLabel: string = "#email";
+    private static passwordLabel: string = "#passwd";
+    private static submitButton: string = "#SubmitLogin";
+
+    public static emailInput(email: string): void {
+        cy.get(Login.emailLabel).type(email);
+    }
+
+    public static passwordInput(password: string): void {
+        cy.get(Login.passwordLabel).type(password);
+    }
+    public static submit(): void {
+        cy.get(Login.submitButton).click();
+  }
```

Comment:
You can make just 1 method that is "Login", where these 3 actions are executed

Reply:
i've already done the changes, but the web site is dead. :(